modules/integration/tests-integration/bpmn/src/test/java/org/wso2/bps/integration/tests/bpmn/JavaServiceTaskDeploymentUndeploymentTestCase.java

```diff
@@ -0,0 +1,94 @@
+package org.wso2.bps.integration.tests.bpmn;
```

Comment:
Added license headers. Configured IDE for license header addition. Fixed in the latest commit.